lms/templates/banned.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block page_content %}
+{% if not current_user.role.is_banned %}
+<script type="text/javascript">
+    window.location.href = "/exercises"
+</script>
+{% else %}
+<div class="banned-user-message">
+    <p style="font-size:30px">Your user is banned due to unproperate usage</p>

```

Comment:
We usually try to avoid inserting inline CSS in pages.

Also, this line probably should be h1 or something.

Try to use wireframing tools to design the page, and then let's talk about how to implement it.